lib/components/AccessAreaList/AccessAreaListItem.tsx

```diff
@@ -23,6 +25,7 @@ export interface AccessAreaListItemProps extends Pick<ListItemProps, 'size' | 'o
 export const AccessAreaListItem = ({
   name,
   icon,
+  iconUrl,
```

Comment:
Do we need to separate these props? I am a bit concerned that having both might create confusion, especially regarding governance. 

Alternatively, could we have something like

```
export type AccessAreaListItemProps =
    | (AccessAreaListItemDefaultProps & { iconURL: string; icon?: never })
    | (AccessAreaListItemDefaultProps & { icon: string; iconURL?: never });
    ```

Reply:
Good suggestion! I'll do that instead 😊